network-api/networkapi/templates/donate/pages/base.html

```diff
@@ -30,7 +30,11 @@
 
 {% block bodyclass %} donate tw-m-0 tw-font-sans tw-font-normal tw-text-base tw-font-normal tw-text-black tw-bg-white tw-scroll-smooth {% endblock %}
 
-{% block donate_banner %}{% endblock %}
+{% block donate_banner %}
```

Comment:
Per request of @kristinashu, she found that the EOY donate banner was not appearing on donate pages, adding this to the base template will fix that!